src/cl/lighthouse/lighthouse_launcher.star

```diff
@@ -130,23 +125,28 @@ def launch(
 
 	beacon_http_port = beacon_service.ports[BEACON_HTTP_PORT_ID]
 
-	# Launch validator node
+	# Launch validator node if we have a keystore
 	beacon_http_url = "http://{0}:{1}".format(beacon_service.ip_address, beacon_http_port.number)
-
-	validator_config = get_validator_config(
-		launcher.genesis_data,
-		image,
-		log_level,
-		beacon_http_url,
-		node_keystore_files,
-		v_min_cpu,
-		v_max_cpu,
-		v_min_mem,
-		v_max_mem,
-		extra_validator_params,
-	)
-
-	validator_service = plan.add_service(validator_node_service_name, validator_config)
+	if node_keystore_files != None:
```

Comment:
can we instantiate an empty validator_service before the if statement? feels cleaner that we assign it and then use it down stream